pulsar-zookeeper-utils/src/main/java/com/yahoo/pulsar/zookeeper/ZooKeeperCache.java

```diff
@@ -74,6 +74,7 @@
     protected final Cache<String, Set<String>> childrenCache;
     protected final Cache<String, Boolean> existsCache;
     protected final OrderedSafeExecutor executor;
+    public static final int cacheTimeOutInSec = 120;
```

Comment:
ZK read operation should complete fairly quickly. If anyway we timeout early, it would just mean permission denied for one particular request but it will be able to catch up later.